components/FaqPage/FaqPage.tsx

```
@@ -0,0 +1,629 @@
+                    <div key={lineIndex} className="mb-1">
+                      {line.startsWith("•") ? (
+                        <span
+                          className="font-spectral font-medium text-[14px] md:text-[18px]"
+                          style={{
+                            fontWeight: 400,
+                            lineHeight: "160%",
+                            letterSpacing: "-5%",
+                          }}
+                          dangerouslySetInnerHTML={{
+                            __html: line
+                              .replace(
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)